assets/css/style.css

```diff
@@ -2277,4 +2278,4 @@ figcaption {
   display: block;
 }
 
-/*# sourceMappingURL=maps/style.css.map */
\ No newline at end of file
+/*# sourceMappingURL=maps/style.css.map */
```

Comment:
Remove this line

Reply:
Is there any influence in the pull-request topic?, or is just something additional to the pull request topic.

Reply:
I am sorry, but I didn't get it.